packages/fxa-auth-server/lib/routes/support.js

```diff
@@ -76,7 +76,7 @@ module.exports = (log, db, config, customs, zendeskClient) => {
           subject,
           requester: {
             email,
-            name: 'Anonymous User',
+            name: email,
```

Comment:
Will this break anything if the user changed their email?

Reply:
We have an existing ticket to push email changes to Zendesk that is lower priority, which would affect this first. But no, this won't break anything.